troubleshooting/cms_barebones_image_boot.md

```diff
@@ -0,0 +1,253 @@
+# Barebones Image Boot Additional Information
+
+This provides additional information and manual steps for reproducing the barebones image boot test.
```

Comment:
This should include a little bit more detail. Maybe something like..

"Verify that the CSM services needed to boot a node are available and working properly. This section describes how the barebonesImageTest script works and how to interpret the results. If the script is unavailable, the manual steps for reproducing the barebones image boot test are provided."